superset-frontend/src/explore/components/controls/DndColumnSelectControl/DndColumnSelect.tsx

```diff
@@ -143,6 +143,7 @@ export function DndColumnSelect(props: DndColumnSelectProps) {
       optionSelector.values.map((column, idx) =>
         isFeatureEnabled(FeatureFlag.ENABLE_DND_WITH_CLICK_UX) ? (
           <ColumnSelectPopoverTrigger
+            key={idx}
```

Comment:
It seems like the `optionSelector` values might change here, meaning using `idx` as the key here might not be a completely safe move. But then, *think* React uses the index by default if another key is not provided, so it's probably doing this already under the hood...

Maybe we can use a better key? Maybe I'm overthinking this?